book/07-git-tools/sections/advanced-merging.asc

```diff
@@ -96,7 +96,7 @@ $ git commit -am 'document the function'
  1 file changed, 1 insertion(+)
 ----
 
-Now we try to merge in our `whitespace` branch and we'll get conflicts because of the whitespace changes.
+Ngayon ay sinusubukan namin na i-merge sa aming `whitespace` na branch at makakuha tayo ng mga salungat dahil sa whitespace na mga pagbabago.
```

Comment:
Change `namin` and `aming` to `natin` and `ating`. This to ensure consistency when dealing when the pronouns such as 'we' in this context.

Reply:
done change